.github/workflows/benchmark.yml

```diff
@@ -43,21 +43,24 @@ on:
     # everday at 10pm UTC
     -   cron: 0 22 * * 0
 
+    workflow_dispatch:  # This enables manual triggering from GitHub UI
+
 jobs:
     benchmark:
         if: >-
-            (contains(github.event.head_commit.message, '[bm]') ||
+            (github.event_name == 'workflow_dispatch' ||
```

Comment:
Unrelated, but I think it's better to trigger via Github UI rather than commit to main.
Didn't know about this before...